docs/reference/config.md

```diff
@@ -0,0 +1,23 @@
+<!--[metadata]>
++++
+title = "config"
+description = "Config validates and view the compose file."
+keywords = ["fig, composition, compose, docker, orchestration, cli, config"]
+[menu.main]
+identifier="config.compose"
+parent = "smn_compose_cli"
++++
+<![end-metadata]-->
+
+# config
+
+```:
+Usage: config [options]
+
+Options:
+-q, --quiet     Only validate the configuration, don't print
+                anything.
+--services      Print the service names, one per line.
+```
+
+Config validates and view the compose file.
```

Comment:
Well… had the same reaction than you, but in the actual usage it's written like this so… didn't want to change :cat: (see https://github.com/docker/compose/blob/master/compose/cli/main.py#L197)